connectors/email/src/main/java/io/camunda/connector/email/outbound/protocols/actions/ImapListEmails.java

```diff
@@ -46,13 +46,7 @@ public record ImapListEmails(
             type = TemplateProperty.PropertyType.Dropdown,
             constraints = @TemplateProperty.PropertyConstraints(notEmpty = true),
             defaultValue = "RECEIVED_DATE",
-            choices = {
-              @TemplateProperty.DropdownPropertyChoice(
-                  label = "Received Date",
-                  value = "RECEIVED_DATE"),
-              @TemplateProperty.DropdownPropertyChoice(label = "Sent Date", value = "SENT_DATE"),
-              @TemplateProperty.DropdownPropertyChoice(label = "Size", value = "SIZE")
-            },
+            enumLabel = @TemplateProperty.EnumLabel,
```

Comment:
Cant this be deducted based on the property type? Like if its an enum do this...